extensions/authentication-jwt/src/__tests__/fixtures/controllers/user.controller.ts

```diff
@@ -95,4 +102,70 @@ export class UserController {
   async whoAmI(): Promise<string> {
     return this.user[securityId];
   }
+  // Routes using refreshtoken
+  @post('/users/refresh/login', {
+    responses: {
+      '200': {
+        description: 'Token',
+        content: {
+          'application/json': {
+            schema: {
+              type: 'object',
+              properties: {
+                accessToken: {
+                  type: 'string',
+                },
+                refreshToken: {
+                  type: 'string',
+                },
+              },
+            },
+          },
+        },
+      },
+    },
+  })
+  @intercept('refresh-token-generate')
```

Comment:
Any reason writing the refresh logic in interceptors? 
E.g. we can expose functions in the refresh token services to do it, and invoke them in the controller, unless we want to do the refresh as a hook for more methods.

See discussion in https://github.com/strongloop/loopback4-example-shopping/pull/537/files#diff-dc782452c9aebb03cbfee4095eb3747cR301

Reply:
@jannyHou Honestly speaking while writing this was not much closer to services, and I thought something like plug gable, so used intercecptors. Well I can create a service, I think which is a broader perspective. 

also before starting the implementation I asked to community here
(https://github.com/strongloop/loopback-next/issues/5187#issuecomment-634691151)

But no one suggested, So begin with the thought.